src/serlo-editor/plugins/rows/editor.tsx

```diff
@@ -2,13 +2,12 @@ import type { RowsProps } from '.'
 import { AllowedChildPlugins } from './allowed-child-plugins-context'
 import { AddRowButtonLarge } from './components/add-row-button-large'
 import { RowEditor } from './components/row-editor'
-import { selectParentPluginType, useAppSelector } from '@/serlo-editor/store'
+import { selectParentPluginType, store } from '@/serlo-editor/store'
 import { EditorPluginType } from '@/serlo-editor-integration/types/editor-plugin-type'
 
 export function RowsEditor({ state, config, id, editable }: RowsProps) {
-  const parentType = useAppSelector((state) =>
-    selectParentPluginType(state, id)
-  )
+  // this should not change so we don't need to use useAppSelector
```

Comment:
Can it not change on drag and drop?

Reply:
Thanks for the question.
If I'm not mistaken we only use it to check if the parent of the rows plugin is the template plugin or the article plugin (basically the main root element). And rows plugins can't be moved so we should be good.

Extended the comment and moved some code to make it a bit clearer.